packages/esm-patient-appointments-app/src/appointments/appointments-base.component.tsx

```diff
@@ -39,7 +41,7 @@ const AppointmentsBase: React.FC<AppointmentsBaseProps> = ({ patientUuid }) => {
       workspaceTitle: t('scheduleAppointment', 'Schedule appointment'),
     });
 
-  if (isLoading) return <DataTableSkeleton role="progressbar" />;
+  if (isLoading) return <DataTableSkeleton role="progressbar" compact={!isTablet} zebra />;
```

Comment:
This would probably read better if we didn't use negation:

```suggestion
  const isDesktop = useLayoutType() === 'small-desktop' || useLayoutType() === 'large-desktop';


  if (isLoading) return <DataTableSkeleton role="progressbar" compact={isDesktop} zebra />;
```